recmetrics/plots.py

```diff
@@ -177,7 +178,7 @@ def MapkPlot(mapk_scores, model_names, k_range):
     plt.show()
 
 
-def ClassDistributionPlot(pred_df, n_bins, threshold=0.5):
+def class_distribution_plot(pred_df, n_bins=150, threshold=0.5, figsize=(10,6), class1_label=None, class0_label=None):
```

Comment:
Can we add the new args, `figsize` ,`class1_label` and `class0_label` to the docstring.